lab_1_classify_by_unigrams/main.py

```diff
@@ -63,6 +122,25 @@ def detect_language(
     :param profile_2: a dictionary of a known profile
     :return: a language
     """
+    if not (
+            isinstance(unknown_profile, dict)
+            and isinstance(profile_1, dict)
+            and isinstance(profile_2, dict)
+    ):
+        return None
+
+    profile_1_metric = compare_profiles(unknown_profile, profile_1)
+    profile_2_metric = compare_profiles(unknown_profile, profile_2)
+
+    if (isinstance(profile_1_metric, float)
+            and isinstance(profile_2_metric, float)
+    ):
```

Comment:
Consider doing an early return here as well

Reply:
i tried but mupy checks didn't work. i'll try again)

Reply:
Did you find a way?